### PR TITLE
Refactor SQLite parameters

### PR DESCRIPTION
**Pull request by quantumagi**

### CODE REVIEW

File: src/Stratis.Features.SQLiteWalletRepository/Tables/DBTable.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Globalization;
+using NBitcoin;
+
+namespace Stratis.Features.SQLiteWalletRepository.Tables
+{
+    internal class DBTable
+    {
+        // Intended to handle string, int, long and decimal.
+        internal static string DBParameter(object prop)
```
Comment (rowandh) [inline src/Stratis.Features.SQLiteWalletRepository/Tables/DBTable.cs]: Could this comment be extended to describe the problem being solved here?

File: src/Stratis.Features.SQLiteWalletRepository/Tables/DBTable.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Globalization;
+using NBitcoin;
+
+namespace Stratis.Features.SQLiteWalletRepository.Tables
+{
+    internal class DBTable
```
Comment (rowandh) [inline src/Stratis.Features.SQLiteWalletRepository/Tables/DBTable.cs]: Is it desirable to require table classes to inherit from this, or could it just be a static class? To me it doesn't make much sense to add inheritance just to access a static method.

File: src/Stratis.Features.SQLiteWalletRepository/Tables/DBTable.cs (hunk 1 above)
Comment (quantumagi) [reply src/Stratis.Features.SQLiteWalletRepository/Tables/DBTable.cs]: Done.

File: src/Stratis.Features.SQLiteWalletRepository/Tables/DBTable.cs (hunk 2 above)
Comment (quantumagi) [reply src/Stratis.Features.SQLiteWalletRepository/Tables/DBTable.cs]: Done.

**Pull request merged**